tf-modules/distribution/main.tf

```diff
@@ -31,11 +33,31 @@ resource "aws_secretsmanager_secret_version" "thin_egress_urs_creds" {
   })
 }
 
+module "tea_map_cache" {
+  prefix = var.prefix
+  source = "../tea-map-cache"
+  lambda_processing_role_arn = var.lambda_processing_role_arn
+  tea_api_url = module.thin_egress_app.internal_api_endpoint
+  tags = var.tags
+  lambda_subnet_ids = var.subnet_ids
+  vpc_id = var.vpc_id
+}
+
+
+data "aws_lambda_invocation" "tea_map_cache" {
+  depends_on = [module.tea_map_cache.lambda_function_name]
+  function_name = module.tea_map_cache.lambda_function_name
+  input = jsonencode({ bucketList = local.tea_buckets,
+                       s3Bucket = var.system_bucket
+                       s3Key = "${var.prefix}/distribution_bucket_map.json"
+  })
+}
+
 module "thin_egress_app" {
-  source = "https://s3.amazonaws.com/asf.public.code/thin-egress-app/tea-terraform-build.74.zip"
+  source = "s3::https://s3.amazonaws.com/cumulus-test-sandbox-private/tea-terraform-manual_build.zip"
```

Comment:
This is awaiting release/review of https://github.com/asfadmin/thin-egress-app/pull/176